tigerpath/static/app/js/tutorial.js

```diff
@@ -26,17 +28,19 @@ function startIntro() {
         If a course can only satisfy one of multiple requirements, it will show up underneath each of those requirements and be grayed out.
         You can click on a grayed out course to make it count for that requirement.
         <br><br>
-        You can also hover over a requirement to read more information about it, and to search for courses that satisfy that requirement!`
+        You can also hover over a requirement to read more information about it, and to search for courses that satisfy that requirement!`,
       },
       {
-        element: '#netid-btn',
-        intro: "If you would like to change your major, add a nickname, or modify other settings, you can click here and then on \"Settings\"."
+        element: '#main-view-tabs',
+        intro:
+          'If you have AP credits, summer courses, or other external credits you want to show up in your requirements, you can switch tabs here and add them.',
       },
       {
-        element: '#help-btn',
-        intro: "If you've missed anything, feel free to view this tutorial at any time by clicking here. We hope you enjoy using TigerPath!"
+        element: '#netid-btn',
+        intro:
+          "Click here to change your major and year in the settings.<br><br>If you've missed anything, you can also view the tutorial here at any time.<br><br>We hope you enjoy using TigerPath!",
```

Comment:
```suggestion
          "Click here to change your major and year in the settings.<br><br>If you've missed anything, you can also view the tutorial here again at any time.<br><br>We hope you enjoy using TigerPath!",
```

Reply:
"here again" sounds a bit weird to me. Is it really necessary to add "again"? I think most users would know they're looking at the tutorial, or they can infer it from the "if you've missed anything". And it doesn't seem like a big deal to me anyway if they view the tutorial and realize they've already seen it

Reply:
Yeah, it's obviously not a big deal. Just thought it's slightly clearer.
If you don't like the "here again" wording you also do
```suggestion
          "Click here to change your major and year in the settings.<br><br>If you've missed anything, you can also view the tutorial again here at any time.<br><br>We hope you enjoy using TigerPath!",
```
Not a big deal either way.

Reply:
Still seems a bit wordy to me. I can't find a way to formulate it to flow nicely and it doesn't seem like an incredibly important piece of information to add, so I'll just leave it as is